event-schema/event_schema.go

```diff
@@ -345,20 +341,24 @@ func (manager *EventSchemaManagerT) handleEvent(writeKey string, event EventT) {
 		}
 
 		if wasOffloaded {
-			manager.reloadModel(offloadedModel)
-			eventModel, ok = manager.eventModelMap[WriteKey(writeKey)][EventType(eventType)][EventIdentifier(eventIdentifier)]
-			if !ok {
-				pkgLogger.Errorf(`[EventSchemas] Failed to reload event +%v, writeKey: %s, eventType: %s, eventIdentifier: %s`, offloadedModel.UUID, writeKey, eventType, eventIdentifier)
-				return
+			err := manager.reloadModel(offloadedModel)
+			if err != nil {
```

Comment:
cannot comment on the logic introduced here

Reply:
Apologies, as per the below comment I thought I had removed this logic with the plan to move it to another PR. I must've messed up with the rebase.

https://github.com/rudderlabs/rudder-server/pull/2064/files#r900043198

I'll remove it (again :sweat:).